### PR TITLE
Add support for cbc.ca

### PR DESCRIPTION
**Pull request by atomicdryad**

### CODE REVIEW

File: youtube_dl/extractor/cbc.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+        #   TP_FEED_DOMAIN:"http://tpfeed.cbc.ca/f/h9dtGB/5akSXx4Ng_Zn?"
+        #   MPX_ACCOUNT_PID:"h9dtGB"
+        tp_feed_domain = "http://tpfeed.cbc.ca/f/h9dtGB/5akSXx4Ng_Zn?"
+        mpx_account_id = "h9dtGB"
+
+        name = url_basename(url)
+
+        webpage = self._download_webpage(url, name)
+        title = unescapeHTML(
+            self._search_regex('<title>\s*(.+?)\s*</title>', webpage, 'title'))
```
Comment (jaimeMF) [inline youtube_dl/extractor/cbc.py]: Doesn't `_html_search_regex`work here?

**Comment by yhager** [comment]
I'm interested in that too , and the tests seem to be working. Any chance this was a travis fluke and can be tried again?

**Comment by yan12125** [comment]
Since #8497 landed, please move changes in ```youtube_dl/extractors/__init__.py``` to ```youtube_dl/extractors/extractors.py```. Check step 5 of the new [developer instructions](https://github.com/rg3/youtube-dl/blob/master/README.md#adding-support-for-a-new-site) for more information.

**Comment by yan12125** [comment]
Since @remitamine has pushed his/her own implementation in 8b85ac3fd95dbc3296801036f43c8d2ebfefcb6f, this PR is no longer necessary.

**Pull request closed without merging**